src/Context/Context.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  FC,
+  createContext,
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+import { Error, States, Todo } from '../types/Todo';
+import { getTodos } from '../api/todos';
+import { getFilteredTodos } from '../helpers';
+
+interface TodoContextType {
+  renderedTodos: Todo[];
+  setRenderedTodos: (value: Todo[]) => void;
+  filteredList: Todo[];
+  selectedOption: string;
+  setSelectedOption: (value: string) => void;
+  errorMessage: Error;
+  setErrorMessage: React.Dispatch<React.SetStateAction<Error>>;
+}
+
+const TodoContext = createContext<TodoContextType>(
+  {
+    renderedTodos: [],
+    setRenderedTodos: () => {},
+    filteredList: [],
+    selectedOption: '',
+    setSelectedOption: () => {},
+    errorMessage: {
+      load: '',
+      add: '',
+      remove: '',
+      title: '',
+      update: '',
+    },
```

Comment:
As I said previously, it's too much in this case I think. It could be just a string with an error message